arm-resources/2014-04-01-preview/swagger/resources.json

```diff
@@ -146,7 +146,8 @@
               "$ref": "#/definitions/DeploymentExtended"
             }
           }
-        }
+        },
+		"x-ms-long-running-operation": true,
```

Comment:
@vivsriaus - Last items in a json array/dictionary do not have a comma. Travis Ci is failing because of the comma after true. Please remove it.